examples/bridge-app/bridge-common/gen/CHIPClustersObjc.h

```diff
@@ -0,0 +1,117 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// THIS FILE IS GENERATED BY ZAP
+
+#ifndef CHIP_CLUSTERS_H
+#define CHIP_CLUSTERS_H
+
+#import <Foundation/Foundation.h>
+
+typedef void (^ResponseHandler)(NSError * _Nullable error, NSDictionary * _Nullable values);
```

Comment:
Maybe use generics here to indicate what the dictionary is expected to be carrying? 
Looks like you want -`NSDictionary<NSString *, id> * _Nullable values` 
It'll help the compiler give us better warnings.

Reply:
There will be a bunch of Objc API updates as a followup to this PR. I'm adding that to my list of proposed changes.